eventstore/options.go

```diff
@@ -147,6 +147,13 @@ func WithDebug(enable bool) StoreOption {
 	}
 }
 
+func WithDatastore(ds ds.Datastore) StoreOption {
```

Comment:
I needed this to leverage an existing store (the one created for the threadservice)